src/components/Checkbox/Checkbox.scss

```diff
@@ -48,6 +48,7 @@ $box-size: 16px;
   height: 12px;
   border: 2px solid $color-white;
   transform: rotate(45deg);
+  backface-visibility: hidden; // required to handle a bug in Chrome
```

Comment:
Well there's something I've never seen before!

Reply:
I've reverted this - I just committed it to the wrong branch by mistake. After spending the weekend writing a blog post about not screwing up with git...

Reply:
Lol yes, I saw that blog post, it was good. I need to chat with you about Dennis Chopper.